docs/docs/creating-dynamically-rendered-navigation.md

```diff
@@ -2,7 +2,176 @@
 title: Creating Dynamically-Rendered Navigation
 ---
 
-This is a stub. Help our community expand it.
 
-Please use the [Gatsby Style Guide](/docs/gatsby-style-guide/) to ensure your
-pull request gets accepted.
+## Creating dynamic navigation in Gatsby
+
+Often at times you will want to be able to easily edit your websites navigation in response to a *change in requirements*. To achieve this, you can use gatsby to dynamically generate your navigation. Where you store the data for your navigation can be anywhere a backend API, CMS, headless CMS or even the filesystem.
```

Comment:
Please avoid words like "easily" ( https://next.gatsbyjs.org/docs/gatsby-style-guide/#avoid-easy-and-simple )

Reply:
Looks like that the header #avoid-easy-and-simple gets pulled behind the top nav bar, so make sure to scroll up a little bit to see the whole paragraph @outofthisworld! Thanks for writing this article!

Reply:
^ and I will be writing a bug report for that right now ;)